components/prism-stylus.js

```diff
@@ -1,6 +1,13 @@
 (function (Prism) {
 	var inside = {
-		'url': /url\((["']?).*?\1\)/i,
+		'comment': {
+			pattern: /(^|[^\\])(?:\/\*[\s\S]*?\*\/|\/\/.*)/,
```

Comment:
Since the `comment` pattern isn't always the first pattern, it should be greedy.